Entities/Characters/Scripts/RunnerDefault.as

```diff
@@ -58,12 +58,6 @@ void onAttach(CBlob@ this, CBlob@ attached, AttachmentPoint @attachedPoint)
 	this.server_DetachFrom( attached ); CRASHES*/
 }
 
-// set the Z back
```

Comment:
Why is this removed?

This appears to cause an issue when leaving a catapult, for instance. The player Z appears to be modified when entering a catapult to visually appear behind it, but with this change, the player will still appear behind the catapult.

Reply:
I reverted the remove. I always forget to test with catapults.